### ISSUE TITLE
question about performance in DocAVQ

### ISSUE THREAD

**Issue by shaoqb** [anisha2102/docvqa#8]
I use your code, the performance on the DocAVQ dataset can only reach 49. Can you provide your trained model？

**Comment by anisha2102** [comment]
I have already provided the Model Weights here: https://github.com/anisha2102/docvqa#model-checkpoints

PS: In the final submission, I used a second model for handling the cases of empty predictions of the model shared above.

**Comment by shaoqb** [comment]
> I have already provided the Model Weights here: https://github.com/anisha2102/docvqa#model-checkpoints
> 
> PS: In the final submission, I used a second model for handling the cases of empty predictions of the model shared above.

second model？ This link has only one model.
I use this model to predict on the docvqa only get 45...

**Comment by shaoqb** [comment]
Can you help me answer the question?

**Comment by anisha2102** [comment]
Hi @shaoqb,

I have not updated the testing and postprocessing scripts for this repository yet - this repository was only meant for sharing the training code so people can play around with it. So it will be difficult to reproduce exact results.

I  will surely try to update the code further when I have time.

Some pointers which might be helpful:
1. Look at the outputs of the model - As the metric is ANLS even mismatches of spaces between prediction and ground-truth can harm your performance. I  had performed some post-processing to tackle this.
2. You can try to retrain the model using the code and pick your best checkpoint to handle outputs for cases where the above model did not give any answer.

**Comment by shaoqb** [comment]
could you update  the testing and postprocessing scripts ？This will help me reproduce the results of docvqa，I will be very grateful to you！
